src/test/resources/comment/CommentsOnCaseExpression.java

```diff
@@ -0,0 +1,19 @@
+public class CommentsOnCaseExpression {
+    public void commentsShouldBeAttachedToCtLiteral(int stage) {
+        String stageStr;
+        boolean fullStatus;
+        switch (stage) {
+            case (1/*org.apache.coyote.Constants.STAGE_PARSE*/):
```

Comment:
Could you improve the test case and a comment before a case, e.g.,`/*org.apache.coyote.Constants.STAGE_PARSE*/1`

Reply:
I added inline comment to add variety.